test/attr/attr_downgrade_exhaustivity.swift

```diff
@@ -0,0 +1,43 @@
+// RUN: %target-typecheck-verify-swift -swift-version 3 %s
+
+enum Runcible {
+  case spoon
+  case hat
+  @_downgrade_exhaustivity_check
```

Comment:
I don't think that's necessary, I just missed the fact it was already conditional on version. Thanks.